api/src/test/java/uk/gov/dhsc/htbhf/card/testhelper/CardDTOTestDataFactory.java

```diff
@@ -14,14 +16,59 @@
 public class CardDTOTestDataFactory {
 
     public static CardDTO aValidCard() {
+        return defaultCardBuilder().build();
+    }
+
+    public static CardDTO aCardWithAddress(AddressDTO address) {
+        return defaultCardBuilder()
+                .address(address)
+                .build();
+    }
+
+    public static CardDTO aCardWithFirstName(String firstName) {
+        return defaultCardBuilder()
+                .firstName(firstName)
+                .build();
+    }
+
+    public static CardDTO aCardWithLastName(String lastName) {
+        return defaultCardBuilder()
+                .lastName(lastName)
+                .build();
+    }
+
+    public static CardDTO aCardWithDateOfBirth(LocalDate dateOfBirth) {
+        return defaultCardBuilder()
+                .dateOfBirth(dateOfBirth)
+                .build();
+    }
+
+    public static CardDTO aCardWithClaimId(String claimId) {
+        return defaultCardBuilder()
+                .claimId(claimId)
+                .build();
+    }
+
+    public static CardDTO aCardWithEmail(String email) {
+        return defaultCardBuilder()
+                .email(email)
+                .build();
+    }
+
+    public static CardDTO aCardWithMobile(String mobile) {
+        return defaultCardBuilder()
+                .mobile(mobile)
+                .build();
+    }
```

Comment:
It's a personal preference, but I would not have created most of these methods, and instead had a method in the factory returning a builder.